targets/TARGET_STM/TARGET_STM32L4/TARGET_STM32L475xG/TARGET_MTB_STM_L475/system_clock.c

```diff
@@ -0,0 +1,370 @@
+/* mbed Microcontroller Library
+* Copyright (c) 2006-2017 ARM Limited
```

Comment:
can you add SPDX identifier for new files

Reply:
Thanks @0xc0170 .. SPDX identifier for all of them or just this file?

Reply:
All new files

Reply:
Ok. 1 more commit coming up!